tests/models/test_models.py

```diff
@@ -5,27 +5,20 @@
 import pytest
 
```

Comment:
comment that 7B models should be tested in test big models. py

Reply:
Comment added